### ISSUE TITLE
Make select RTL'able

### ISSUE THREAD

**Issue by Confidenceman02** [Confidenceman02/elm-select#39]
# Context
RTL languages currently do not have a  good experience with elm-select which is unacceptable.

# Work to be done
- RTL support should be baked in instead of an option you can switch on.
- Use logical properties where possible to aid in RTL support. (Low hanging fruit)

**Comment by Confidenceman02** [comment]
Released in [12.1.1](https://package.elm-lang.org/packages/Confidenceman02/elm-select/12.1.1/)
